examples/modules/05b-partial-import/example.effekt

```diff
@@ -0,0 +1,6 @@
+import hello.world.{greet goodbye}
```

Comment:
Since we borrow a lot of Syntax from Scala, we could think about also using a comma to separate different imports. That is: 

```suggestion
import hello.world.{greet, goodbye}
```